src/documentation/developers/grid_deployment/grid_deployment_full_vm.md

```diff
@@ -71,6 +71,10 @@ We show the steps to prepare the VM to run the network instance.
 
 If you are deploying on testnet or devnet, simply replace `mainnet` by the proper network in the following lines.
 
+- Set the prerequisites
+  ```
+  apt update && apt install -y git 
```

Comment:
You also need to install nano

Reply:
And ufw

Reply:
good catch. added. resolving this conversation.